src/interface/src/app/plan/plan-summary/saved-scenarios/saved-scenarios.component.ts

```diff
@@ -56,32 +56,23 @@ export class SavedScenariosComponent implements OnInit {
       });
   }
 
-  viewScenario(id?: string): void {
-    if (!id) {
-      id = this.scenarios.find((scenario) => scenario.selected)?.id;
-    }
-    this.router.navigate(['scenario', id], { relativeTo: this.route });
+  viewScenario(): void {
+    this.router.navigate(['config', this.highlightedId], {
+      relativeTo: this.route,
+    });
   }
 
-  showDeleteButton(): boolean {
-    return this.scenarios.filter((scenario) => scenario.selected).length > 0;
-  }
-
-  showViewButton(): boolean {
-    return this.scenarios.filter((scenario) => scenario.selected).length === 1;
+  highlightScenario(id: string): void {
+    this.highlightedId = id;
   }
 
   deleteSelectedScenarios(): void {
     this.planService
-      .deleteScenarios(
-        this.scenarios
-          .filter((scenario) => scenario.selected)
-          .map((scenario) => scenario.id)
-      )
+      .deleteScenarios([this.highlightedId])
       .subscribe({
         next: (deletedIds) => {
           this.snackbar.open(
-            `Deleted ${deletedIds.length} scenario${
+            `Deleted scenario${
```

Comment:
I don't think it makes sense to have a case where we don't get any deletedIds. Can we update this code?
Also, we should probably reset the `this.highlightedI` after deletion

Reply:
Sorry, where's the case where we don't get any deletedId's here? I kept the logic to make the deleted message plural if multiple scenarios are deleted for the future in case y'all re-implement that (currently you can only delete one at a time). I'm not seeing here where it's doing anything if there's no deletedId's

Reply:
Sorry you're right I read the line below wrong!

Reply:
No worries! I'll add a clarifying comment there since it's a bit confusing (especially since bulk deletion isn't currently implemented)